app/main.py

```diff
@@ -1,3 +1,72 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float = 0) -> None:
```

Comment:
Indicate that `km` could be `int` as well. Do not provide default values

Reply:
This was not resolved